src/js/content_scripts/input-mapper.js

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
+/* eslint-env browser */
+
+(function (fluid) {
+    "use strict";
+
+    var gamepad = fluid.registerNamespace("gamepad");
+    fluid.registerNamespace("gamepad.inputMapperUtils");
+
+    fluid.defaults("gamepad.inputMapper", {
+        gradeNames: ["gamepad.navigator"],
+        model: {
+            actionMap: {
+                buttons: {
+                    "0": "vacant",
+                    "1": "vacant",
+                    "2": "vacant",
+                    "3": "vacant",
+                    "4": "vacant",
+                    "5": "vacant",
+                    "6": "vacant",
+                    "7": "vacant",
+                    "8": "vacant",
+                    "9": "vacant",
+                    "10": "vacant",
+                    "11": "vacant",
+                    "12": "scrollUp",
+                    "13": "scrollDown",
+                    "14": "scrollLeft",
+                    "15": "scrollRight",
+                    "16": "vacant"
+                },
+                axes: {
+                    "0": "scrollHorizontally",
+                    "1": "scrollVertically",
+                    "2": "vacant",
+                    "3": "vacant"
```

Comment:
Although the standard reference controller in the [Gamepad API standard](https://www.w3.org/TR/gamepad/) has 4 axes, there are controllers with more (or less axes).  This is one way you might end up trying to launch `undefined` as a function using your current approach.  I think for now simply confirming that you were able to find an action mapped to the input is enough to prevent this type of problem.